apps/shade/src/components/ui/data-list.stories.tsx

```
@@ -0,0 +1,91 @@
+    component: DataList,
+    tags: ['autodocs']
+} satisfies Meta<typeof DataList>;
+
+export default meta;
+type Story = StoryObj<typeof DataList>;
+
+export const Default: Story = {
+    args: {
+        className: '',
+        children: [
+            <DataListHeader>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)